### PR TITLE
Fixed iOS podspec for ReactNative 0.68.3+ support

### PR DESCRIPTION
**Pull request by ddolheguy**
This change fixes the PODSpec file in iOS to support the new changes in ReactNative 0.68.3+

### CODE REVIEW

**Review by warrendeleon — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-jw-media-player",
-  "version": "0.2.26",
+  "version": "0.2.3",
```
Comment (archcorsair) [inline package.json:3]: This should be `"version": "0.2.27"`

**Review by archcorsair — CHANGES_REQUESTED** [review]
Please restore [`react-native.config.js`](https://github.com/chaimPaneth/react-native-jw-media-player/blob/master/react-native.config.js)

File: package.json (hunk 1 above)
Comment (ddolheguy) [reply package.json]: @archcorsair Thanks for pointing this out, all updated.

**Comment by archcorsair** [comment]
Please restore this file: [`react-native.config.js`](https://github.com/chaimPaneth/react-native-jw-media-player/blob/master/react-native.config.js)

It appears that you unintentionally deleted it

**Comment by ddolheguy** [comment]
@archcorsair actually this was deleted because the file is no longer required as per the ReactNative spec and was actually incorrect.  The iOS reference within the file needed to be removed due to linking issues being caused.  This has been fully tested on Android and iOS with the library correctly linking and working.

**Comment by archcorsair** [comment]
> @archcorsair actually this was deleted because the file is no longer required as per the ReactNative spec and was actually incorrect. The iOS reference within the file needed to be removed due to linking issues being caused. This has been fully tested on Android and iOS with the library correctly linking and working.

Awesome! Thanks for clarifying this. I think this looks fine to me, before I approve anything I'd like to also run this by @chaimPaneth

**Comment by chaimPaneth** [comment]
Hi @ddolheguy  @archcorsair 

The changes looks fine though I have a few projects running react-native 0.68.5 and I had no issues installing, what is the purpose of those changes, what are the errors when trying to install.

**Comment by archcorsair** [comment]
> Hi @ddolheguy @archcorsair
> 
> The changes looks fine though I have a few projects running react-native 0.68.5 and I had no issues installing, what is the purpose of those changes, what are the errors when trying to install.

I can confirm that I had issues after upgrading above RN version `0.69.x`

**Comment by ddolheguy** [comment]
@chaimPaneth This is the thread which helps explain the issue and it was actually @archcorsair who advised the fix https://github.com/chaimPaneth/react-native-jw-media-player/issues/233.

The issue might not be in the 0.68.x range but I'm running 0.70.5 and it seems people in the 0.69+ might have having this issue as well.

**Comment by chaimPaneth** [comment]
Ah I see. I'm merging and releasing fixed build

**Comment by chaimPaneth** [comment]
Released 0.2.27

**Comment by warrendeleon** [comment]
@ddolheguy can we talk? 7510084239

**Pull request merged**